File: packages/cli/tests/commands/template/fixtures/start-cmd/expected/typescript/build.prod.ts
import { Template, defaultBuildLogger } from 'e2b'
import { template } from './template'

async function main() {
  await Template.build(template, {
    alias: 'start-cmd',
    cpuCount: 2,
    memoryMB: 1024,
    onBuildLogs: defaultBuildLogger(),
  });
}

main().catch(console.error);

File: packages/cli/tests/commands/template/fixtures/start-cmd/expected/python-async/build_prod.py
import asyncio
from e2b import AsyncTemplate, default_build_logger
from .template import template


async def main():
    await AsyncTemplate.build(
        template,
        alias="start-cmd",
        cpu_count=2,
        memory_mb=1024,
        on_build_logs=default_build_logger(),
    )


if __name__ == "__main__":
    asyncio.run(main())